### ISSUE TITLE
Visual Studio Solution Fails

### ISSUE THREAD

**Issue by HardlineStudios** [GaijinEntertainment/DagorEngine#97]
I can't seem to get the .SLN files to load correctly. Complains about missing subproject paths. 

Is there another way to load up the code base into Visual Studio?

**Comment by NicSavichev** [comment]
No, there are no other ways to load code to Visual Studio. We are working on restoring VS support (was suddenly removed on 2024_11_05 update)

**Comment by JailbreakPapa** [comment]
@NicSavichev Any Update on this?
